browser/src/map/handler/Map.Keyboard.js

```diff
@@ -456,20 +456,6 @@ window.L.Map.Keyboard = window.L.Handler.extend({
 				this._map._docLayer._preview.partsFocused = false;
 			}
 		}
-		else if (this._isCtrlKey(ev) && ev.keyCode === this.keyCodes.S) {
-			// Save only when not read-only and when HideSaveOption is false.
-			if (!this._map.isReadOnlyMode() && !this._map['wopi'].HideSaveOption) {
-				this._map.fire('postMessage', {msgId: 'UI_Save', args: { source: 'keyboard' }});
```

Comment:
please notice we used 'keyboard' as a source here, in dispatcher we use 'toolbar', I'm not sure if it is important. Did you check?

Also there is `!this._map['wopi'].HideSaveOption` missing in dispatch command. Possibly it is important? to not allow "readonly sessions" which are not marked as `readonly UI` , if that possible...

dispatch action:
https://github.com/CollaboraOnline/online/blob/master/browser/src/docdispatcher.ts#L25